### PR TITLE
Celery stats for active, pending and done tasks.

### PR DESCRIPTION
**Pull request by ilovepjs**

### CODE REVIEW

File: python/web/klee_admin/views.py [hunk 1]
```diff
@@ -58,3 +59,15 @@ def worker_list(request):
...
+
+@group_required("admin")
+def task_list(request, type='active'):
+    task_map = {
+        'active': klee_tasks.active_tasks(),
+        'waiting': klee_tasks.all_waiting_tasks(),
+        'done': klee_tasks.done_tasks()
+    }
+
+    attrs = {'tasks': task_map[type], 'page': type}
```
Comment (ains) [inline python/web/klee_admin/views.py]: doesn't this explode if the task is not in the map?, try .get with a default  if you want to use a dict

File: python/web/klee_admin/klee_tasks.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            task = {
+                'id': t['id'],
+                'mach': mach
+            }
+            get_db_attrs(task)
+            all_tasks.append(task)
+    return all_tasks
+
+
+def get_db_attrs(task):
```
Comment (ains) [inline python/web/klee_admin/klee_tasks.py:75]: can we make this function non-side effecting, as in, return a new task with the attributes added

Comment (ilovepjs) [inline python/web/klee_admin/klee_tasks.py:75]: don't see why not

File: python/web/klee_admin/views.py (hunk 1 above)
Comment (ilovepjs) [inline python/web/klee_admin/views.py]: type already get's a default 
type='active'

File: python/web/klee_web/views.py [hunk 3]
```diff
@@ -39,15 +40,35 @@ def submit_job(request):
...
-        return HttpResponse('Ok!')
+    type = request.POST.get('type')
+    channel = request.POST.get('channel')
+    notify(
+        channel,
+        type,
+        request.POST.get('data')
+    )
+    if type == 'job_complete' or type == 'job_failed':
+        task = Task.objects.get(task_id=request.POST.get('channel'))
```
Comment (ains) [inline python/web/klee_web/views.py]: you can use the variable channel

File: python/web/klee_admin/views.py (hunk 1 above)
Comment (ains) [inline python/web/klee_admin/views.py]: Defaults are only used if they type nothing, what if I type "elephant", the whole thing blows?

Comment (ilovepjs) [inline python/web/klee_admin/views.py]: that is a good point, that case got lost in the merge will switch

File: python/web/klee_admin/klee_tasks.py (hunk 2 above)
Comment (ilovepjs) [inline python/web/klee_admin/klee_tasks.py:75]: wait wat do you mean, it's already creating a new task or do you want get_db_attrs to not be side affecting

Comment (ains) [inline python/web/klee_admin/klee_tasks.py:75]: get_db_attrs modifies the task(dictionary) passed into it and returns nothing, i'd rather it took in a dictionary and gave back a new dictionary that has the added attributes

Comment (ilovepjs) [inline python/web/klee_admin/klee_tasks.py:75]: gotcha, happy with it not recopying the old dictionary or want it to create completely new tasks

**Pull request merged**